cmd/bacalhau/utils.go

```diff
@@ -102,6 +103,34 @@ func ReverseList(s []string) []string {
 	return s
 }
 
+//nolint:unused,deadcode
+func capture() func() (string, error) {
```

Comment:
maybe add a short descriptive comment to this func? 😬

Reply:
yes added a comment

Reply:
well done thanks!